test/driver/exception_handling/whole_file_multiple_errors.ml

```diff
@@ -1,16 +1,18 @@
 open Ppxlib
 
 let () =
-  Driver.V2.(
-    register_transformation
-      ~impl:(fun ctxt str ->
-        let loc =
-          match str with
-          | [] -> Location.in_file (Expansion_context.Base.input_name ctxt)
-          | hd :: _ -> hd.pstr_loc
-        in
-        Location.raise_errorf ~loc "A located error in a whole file transform")
-      "a_raise_exc")
+  let instrument =
+    let transformation ctxt str =
+      let loc =
+        match str with
+        | [] -> Location.in_file (Expansion_context.Base.input_name ctxt)
+        | hd :: _ -> hd.pstr_loc
+      in
+      Location.raise_errorf ~loc "SHOULD APPEAR FIRST"
```

Comment:
Maybe a sentence that explains at which stage the error happens would make it more informative

```suggestion
      Location.raise_errorf ~loc "Raising an error during the first instrumentation phase"
```
and similarly for the other errors!